action.yml

```diff
@@ -11,6 +11,9 @@ inputs:
     image:
         description: Name of Container Image
         required: true
+    commit-hash:
+        description: Commit Hash
+        required: true
```

Comment:
If below works, we doesn't need to define it here
https://github.com/Azure/aca-preview/pull/34/files#r971860836